WordPress/src/main/java/org/wordpress/android/util/PerAppLocaleManager.kt

```diff
@@ -31,6 +41,8 @@ class PerAppLocaleManager @Inject constructor(
 
     fun getCurrentLocaleDisplayName(): String = getCurrentLocale().displayName
 
+    fun getCurrentLocaleLanguageCode(): String = getCurrentLocale().language
```

Comment:
Minor (🔍): Consider making this a private function as it is only called from within this manager class.